pleskdistup/common/tests/versiontests.py

```diff
@@ -34,6 +34,9 @@ def test_kernel_start_with_prefix(self):
     def test_kernel_start_with_plus_prefix(self):
         self._check_parse("kernel-plus-3.10.0-327.36.3.el7.centos.plus.x86_64", "3.10.0-327.36.3.el7.centos.plus.x86_64")
 
+    def test_kernel_with_underline(self):
+        self._check_parse("kernel-3.14.43_1-2.x86_64", "3.14.43-1.2.x86_64")
```

Comment:
The versions are getting convoluted. I propose to test components of `KernelVersion` separately. That is, in addition to testing string representation, also test that `major`, `minor`, etc. are correct.

Reply:
> The versions are getting convoluted.
Yep. Unfortunately, I don't think we could do much about it.
Let's add some test here as you proposed.